### PR TITLE
Tutorial: update Building a Graph Convolutional Network tutorial

### PR DESCRIPTION
**Pull request by cylinbao**
Few updates on the Building a Graph Convolutional Network tutorial

* Add bias in the GCN layer
* Load pretrained model exported from DGL and verify model accuracy
* use time_evaluator to measure the runtime

Please review
@ZihengJiang, @tmoreau89

### CODE REVIEW

**Comment by tmoreau89** [comment]
Adding @yuluny2 to also review the PR.

File: tutorials/frontend/build_gcn.py [hunk 1]
```diff
@@ -81,47 +90,48 @@ def GraphConv(
     norm: relay.Expr,
     Norm passed to this layer to normalize features before and after Convolution.
 
+    bias: bool
+    Set bias to True to add bias when doing gcn layer
```
Comment (tmoreau89) [inline tutorials/frontend/build_gcn.py]: capitalize GCN

File: tutorials/frontend/build_gcn.py [hunk 2]
```diff
@@ -240,43 +240,42 @@ def load_dataset(dataset="cora"):
...
-from tvm.contrib import graph_runtime
-import time
 
-# Set up weights. You can modify this part and use your own trained weights.
-params['in_weight'] = np.ones((input_dim, hidden_dim), dtype='float32')
-params['out_weight'] = np.ones((hidden_dim, num_classes), dtype='float32')
-for i in range(num_hidden):
-    params["%s_weight"%(str(i))] = np.ones((hidden_dim, hidden_dim), dtype='float32')
+# Download pretrained GCN model
+model_url = "https://homes.cs.washington.edu/~cyulin/media/gcn_%s.pickle"%(dataset)
```
Comment (tmoreau89) [inline tutorials/frontend/build_gcn.py]: How was this pickle file generated? Was it trained manually?

File: tutorials/frontend/build_gcn.py [hunk 3]
```diff
@@ -240,43 +240,42 @@ def load_dataset(dataset="cora"):
...
+# Run the model for one time and test for accuracy
+m.run()
+outval = m.get_output(0).asnumpy()
+pred = outval.argmax(axis=1)
+accuracy = ((pred == labels) * test_mask).sum() / test_mask.sum()
+print("Test accuracy {:.2%}".format(accuracy))
+
+# Evaluate the runtime
+print("Evaluate inference time cost...")
+timer = m.module.time_evaluator("run", ctx, number=1, repeat=10)
```
Comment (tmoreau89) [inline tutorials/frontend/build_gcn.py]: Increase the number

File: tutorials/frontend/build_gcn.py (hunk 2 above)
Comment (cylinbao) [reply tutorials/frontend/build_gcn.py]: It's trained with this example DGL pytorch code
https://github.com/dmlc/dgl/blob/master/examples/pytorch/gcn/train.py
I can add the source to the comment.

**Comment by yuluny2** [comment]
lgtm

**Comment by cylinbao** [comment]
Thank @tmoreau89 for the comments.
There is no stable GPU scheduling yet, so we can't show the GPU version.

**Comment by cylinbao** [comment]
@tmoreau89 
In my personal testing, TVM runs faster than MxNet but has similar speed with PyTorch.
Also, the inference time is very short (under 30ms) for each iteration.
I'm not sure whether it's fair to show the runtime comparison.

**Comment by ZihengJiang** [comment]
@cylinbao better to change the previous performance claim

**Comment by tmoreau89** [comment]
@cylinbao thanks. If you can just submit a correctness check that will ensure that our computation is correct. Let's leave GPU support and performance comparison to a future PR.

File: tutorials/frontend/build_gcn.py (hunk 2 above)
Comment (tmoreau89) [reply tutorials/frontend/build_gcn.py]: Yes, that would be good to make the tutorial easy to reproduce/extend

**Comment by cylinbao** [comment]
@tmoreau89 
Add the checking with DGL-PyTorch.

@ZihengJiang @yuluny2 
You may want to review again. Thanks!

**Comment by tmoreau89** [comment]
Thanks looks good, please address the comments above.

**Comment by cylinbao** [comment]
I think all the comments have been addressed!

**Comment by tmoreau89** [comment]
Thanks, indeed they've been addressed (I resolved them). Will wait on checks to pass before merging it in!

**Comment by tmoreau89** [comment]
Looks like the CIs failed, can you investigate?

**Comment by cylinbao** [comment]
It seems like the DGL version is too old in the CL.
Is there a way we can update the DGL version to 0.4?

**Comment by cylinbao** [comment]
Add one commit to handle the DGL version issue in the code.

**Comment by tmoreau89** [comment]
Thank you for the update @cylinbao !
